packages/react-components/src/button/src/IconButton.css

```diff
@@ -0,0 +1,29 @@
+/* ICON BUTTON */
+.o-ui-icon-button {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: max-content;
+    height: max-content;
+}
+
+/* SIZES | ROUNDED */
+.o-ui-icon-button.o-ui-button-rounded.o-ui-button-2xs {
```

Comment:
Do you really need to `.o-ui-icon-button` here? (and on the subsequent selectors?)